Index: db.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- db.py	(date 1413538074000)
+++ db.py	(revision )
@@ -5,7 +5,7 @@
 
 class DB(object):
 
-    conn = MySQLdb.connect(user="root", db="blog", passwd=yxc, host="localhost")
+    conn = MySQLdb.connect(user="root", db="blog", passwd="yxc", host="localhost")
     cursor = conn.cursor()
 
     def __init__(self):
@@ -15,7 +15,6 @@
         self.bookInfo = bookInfo
         if self.bookInfo.get("book_name") & self.bookInfo.get("book_isbn"):
             if self.bookInfo.get("book_author"):
-               bookInfo = [self.bookInfo.get("book_name"), self.bookInfo.get("book_isbn"),\
+               bookInfo = [self.bookInfo.get("book_name"), self.bookInfo.get("book_isbn"), \
                            self.bookInfo.get("book_author")]
                self.cursor.execute("insert into bookInfo values %s" % bookInfo)
-
